habitat-lab/habitat/gym/gym_wrapper.py

```diff
@@ -249,23 +254,15 @@ def step(self, action: Union[np.ndarray, int]):
 
     @property
     def number_of_episodes(self) -> int:
-        return self._env.number_of_episodes
-
-    def current_episode(self, all_info: bool = False) -> int:
-        """
-        Returns the current episode of the environment.
-        :param all_info: If true, all of the information in the episode
-        will be provided. Otherwise, only episode_id and scene_id will
-        be included
-        :return: The BaseEpisode object for the current episode
-        """
-        return self._env.current_episode(all_info)
+        return self.unwrapped.number_of_episodes
+
+    def current_episode(self, all_info: bool = False) -> "BaseEpisode":
+        return self.unwrapped.current_episode(all_info)
 
     def _direct_hab_step(self, action: Union[int, str, Dict[str, Any]]):
-        obs, reward, done, info = self._env.step(action=action)
+        obs, reward, done, info = self.env.step(action=action)
```

Comment:
why sometimes use .unwrapped and sometimes .env ? I think we should be consistent.

Reply:
My bad, I'll correct this

Reply:
Done